zammad/templates/deployment-sidecars.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.sidecars }}
```

Comment:
the naming seems off.
a sidecar would be a container running in the same pod.

Reply:
Do you have a better suggestion? It's a POD running in the same cluster, with access to the volumes etc.
I used it to keep it compatible in the `values.yaml`.

Reply:
With this MR, there is no need any more to run in the same POD to get access to data.

Reply:
Sidecars might be used for other stuff than accesing zammad data (for example adding external auth).
Therfore adding an deployment would break current functionality.
Depending on the usecase a sidecar, volume & volumemount config for every single deployment might be needed.

Reply:
Thanks for the explanation. I moved the sidecar settings now to be specific to the deployments in 47a8fdd3dd7d953ed5a7468eb0ad8098a5e15b15, can you check please?